concent_api/core/tests/test_integration_force_payment.py

```diff
@@ -181,6 +183,69 @@ def test_provider_send_force_payment_with_subtask_results_accepted_where_ethereu
         )
         self._assert_stored_message_counter_not_increased()
 
+    def test_that_if_provider_sends_force_payment_with_two_different_providers_ethereum_accounts_concent_should_refuse(self):
+        """
+        Expected message exchange:
+        Provider  -> Concent:    ForcePayment
+        Concent   -> Provider:   ServiceRefused
+        """
+        report_computed_task1 = self._get_deserialized_report_computed_task(
+            timestamp="2018-02-05 10:00:05",
+            task_to_compute=self._get_deserialized_task_to_compute(
+                timestamp="2018-02-05 10:00:00",
+                deadline="2018-02-05 10:00:10",
+                subtask_id=self._get_uuid('1'),
+                price=15000,
```

Comment:
Please limit the call only to required attributes for this test. For example price is by default `1` and doesn't need to be set to anything extra for this test. It makes it more complicated. Same for other messages.

Reply:
:+1: done